src/components/Form.js

```diff
@@ -267,7 +267,7 @@ class Form extends React.Component {
                         keyboardShouldPersistTaps="handled"
                         ref={el => this.form = el}
                     >
-                        <View style={[this.props.style, safeAreaPaddingBottomStyle]}>
+                        <FormSubmit style={[this.props.style, safeAreaPaddingBottomStyle]} onSubmit={this.submit}>
```

Comment:
I'm seeing the console error below and it seems to be cause by `this.props.style` being an array. We should spread it and also add the style propTypes to Form, as well as a default empty array prop.

```suggestion
                        <FormSubmit style={[...this.props.style, safeAreaPaddingBottomStyle]} onSubmit={this.submit}>
```
![Screenshot 2023-01-24 at 2 31 41 PM](https://user-images.githubusercontent.com/22219519/214424947-86d9c067-68c9-46e8-b84d-7071c1145286.png)

Reply:
I think `style={[...this.props.style, safeAreaPaddingBottomStyle]}` will cause an issue, when the style is an object, and we are trying to spread it.

Reply:
We can fix it with `style={StyleSheet.flatten([this.props.style, safeAreaPaddingBottomStyle])}`

Reply:
Makes sense! Thanks!